versions.lock

```diff
@@ -16,15 +16,15 @@ com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava (1 c
 com.google.j2objc:j2objc-annotations:1.1 (2 constraints: 8f167910)
 com.palantir.safe-logging:preconditions:1.9.0 (2 constraints: ae11e6f9)
 com.palantir.safe-logging:safe-logging:1.9.0 (3 constraints: b3222d22)
-com.palantir.tracing:tracing:2.15.0 (1 constraints: 3a053e3b)
-com.palantir.tracing:tracing-api:2.15.0 (1 constraints: d00c0e16)
+com.palantir.tracing:tracing:2.16.0 (1 constraints: 3b05413b)
```

Comment:
I'm surprised the `CheckReturnValue` annotation on `startSpan(String)` didn't cause build failures

Reply:
I think `generateErrorPronePatch` has string value `"false"` which evaluates to true-ish

Reply:
yeah 😢 . created https://github.com/palantir/tritium/pull/293 to fix-up error-prone checks